### PR TITLE
Basic app logic

### PR DESCRIPTION
**Pull request by Sandr0x00**
We now can download and unzip files from the web.
We now can access the DB and insert some data.

### CODE REVIEW

File: app/DBConnection.js [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+  });
+};
+
+module.exports.writeRelationships = function(relationJSON) {
+  connect().then(() => {
+    var relationships = context.component('models').module('relationships');
+    //var relationshipDescription = context.component('models').module('relationshipDescriptions');
+    var relationshipEntities = context.component('models').module('relationshipEntities');
+    //var relationshipOccurrences = context.component('models').module('relationshipOccurrences');
+    //var relationshipTypes = context.component('models').module('relationshipTypes');
```
Comment (Henni) [inline app/DBConnection.js]: code comments... Eew

**Review by simonzachau — DISMISSED** [review]
please flatten the promise chain in the WET-Parser

File: app.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+var contentRel = utils.getFileContent('output/relationships.json');
+
+/*content.then(data => {
+  dbConnection.writeEvents(JSON.parse(data));
+});*/
+contentRel.then(data => {
+  dbConnection.writeRelationships(JSON.parse(data));
+});
+
+var wetFileParser = require('./app/WETParser.js');
```
Comment (Henni) [inline app.js]: move to the top + `const`

File: app/DBConnection.js [hunk 3]
```diff
@@ -0,0 +1,95 @@
+'use strict';
+
+const apiLocation = "../api/";
+const api = require(apiLocation + "database.js");
```
Comment (Henni) [inline app/DBConnection.js]: Why store the `apiLocation` in a separate variable? It isn't reused and probably won't change in the future.

File: app/DBConnection.js [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        }).then(() => {
+          return relationships.create({
+            'confidence': relation.quality,
+            'subjectId': subjectId,
+            'objectId': objectId,
+            'relation': relation.relation
+          });
+        }).then(d => {
+          if (d) {
+            //relationshipId = d.id;
```
Comment (Henni) [inline app/DBConnection.js]: what is happening here?

File: config/index.js [hunk 5]
```diff
@@ -1,7 +1,53 @@
...
+  },
+  "db": {
+    "host": "localhost",
+    "port": "5432"
+  },
+  "volume": {
+    "input": "",
+    "output": ""
+  },
+  "algorithmns": [
```
Comment (Henni) [inline config/index.js]: typo algorithm~~n~~s

**Review by Henni — CHANGES_REQUESTED** [review]
First Review, but not yet tested locally

**Review by Henni — APPROVED** [review]

**Comment by Henni** [comment]
@simonzachau are you ok with merging?

**Pull request merged**